src/extension.ts

```diff
@@ -68,12 +68,15 @@ async function pickDevice(): Promise<void> {
         // user canceled
         return;
     }
+    const statusBarMessage = new StatusBarProgressionMessage(`Connecting to device ${device.name}...`);

```

Comment:
Saying `device` is redundant (and I don't have much room in the statusbar in my MacBook).  ``` `Connecting to ${device.name}...` ``` is sufficient.

Reply:
Also, let's use `withProgress` here so we get a spinner.